### ISSUE TITLE
Implement a hamburger menu

### ISSUE THREAD

**Issue by sajvanwijk** [Ycreak/OpensourceClassicCommentary#240]
We need a hamburger menu to place all nonessential features for the playground in.

### List buttons that should go to the playground here:
- [ ] Load/Save?
- [ ] Collaboration?
- [ ] Shortcut to playground settings?

**Comment by Ycreak** [comment]
Yes! And do we implement this like excalidraw: here they show the icon followed by the text, which looks very pretty (and is something i cant implement with my lacking css skillz).

**Comment by Ycreak** [comment]
Maybe we can add this to the menu that appears when you click a fragment? something like that?
